src/transports/dash/low_latency_segment_loader.ts

```diff
@@ -62,7 +62,13 @@ export default function lowLatencySegmentLoader(
     const completeChunks = res[0];
     partialChunk = res[1];
     if (completeChunks !== null) {
-      callbacks.onNewChunk(completeChunks);
+      if (content.type === "text") {
```

Comment:
Though I understand why we do that, I find it weird to have a different logic that only make sense for another part of the code here.
Shouldn't we do this for all sub-segments?

Reply:
Isn't it an optimization to concatenate the segments for the video?